lib/ansible/template/native_helpers.py

```diff
@@ -102,6 +95,10 @@ def ansible_concat(nodes, convert_data, variable_start_string):
     return out
 
 
+def ansible_concat(nodes):
+    return ''.join([to_text(_fail_on_undefined(v)) for v in nodes])
```

Comment:
I think it would be helpful to give docstrings for these functions.  I just spent several minutes jumping back and forth between here and where they were being used to figure out why there were 3 now